neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -1,53 +1,38 @@
 using Neo.Cryptography;
 using Neo.IO;
-using Neo.IO.Caching;
 using Neo.IO.Json;
-using Neo.Ledger;
 using Neo.Persistence;
 using Neo.SmartContract;
+using Neo.SmartContract.Native;
 using Neo.VM;
+using Neo.Wallets;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Text;
 
 namespace Neo.Network.P2P.Payloads
 {
-    public abstract class Transaction : IEquatable<Transaction>, IInventory
+    public sealed class Transaction : IEquatable<Transaction>, IInventory
     {
         public const int MaxTransactionSize = 102400;
         /// <summary>
         /// Maximum number of attributes that can be contained within a transaction
```

Comment:
We should change the comment
```suggestion
        /// Maximum number of cosigners that can be contained within a transaction
```